postman/schemas/transfer.yml

```diff
@@ -11,6 +11,665 @@ servers:
+          description: See [country codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3).
+          schema:
+            "$ref": "#/components/schemas/threeUppercaseChars"
+        - name: currencyISO3Code
+          in: query
+          required: true
+          description: See [currency codes](https://en.wikipedia.org/wiki/ISO_4217).
+          schema:
+            "$ref": "#/components/schemas/threeUppercaseChars"
+        - name: transferMethod
+          in: query
+          required: true
+          schema:
+            "$ref": "#/components/schemas/transferMethods"
+        - name: userType
+          in: query
+          required: true
+          schema:
+            "$ref": "#/components/schemas/userTypes"
+        - name: category
+          in: query
+          required: true
+          schema:
+            "$ref": "#/components/schemas/accountFieldCategories"
+      responses:
+        '200':
+          description: Get Account Fields
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  fieldSets:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        fieldSetId:
+                          type: string
+                          example: BANK_DETAILS
+                        fieldSetName:
+                          type: string
+                          example: Bank Details
+                        fields:
+                          type: array
+                          items:
+                            type: object
+                            properties:
+                              dependencies:
+                                type: object
+                                properties: {}
+                              fieldId:
+                                type: string
+                                example: CARD_ID
+                              fieldType:
+                                type: string
+                                example: TEXT
+                              hintText:
+                                type: string
+                                example: Card Id
+                              isRequired:
+                                type: boolean
+                                example: true
+                              name:
+                                type: string
+                                example: Card Id
+                              placeholderText:
+                                type: string
+                                example: Card Id
+                              visibility:
+                                type: string
+                                example: SHOW_AND_REQUIRED
+              examples:
+                Get Account Fields:
+                  value:
+                    fieldSets:
+                      - fieldSetId: BANK_DETAILS
```

Comment:
Our examples should make sense. BANK_DETAILS as a field set with "CARD_ID" as the field is confusing